instrumentation/openai/openai-java-1.1/library/src/main/java/io/opentelemetry/instrumentation/openai/v1_1/ChatCompletionEventsHelper.java

```diff
@@ -29,14 +29,16 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
-import javax.annotation.Nullable;
 
 final class ChatCompletionEventsHelper {
 
   private static final AttributeKey<String> EVENT_NAME = stringKey("event.name");
 
   public static void emitPromptLogEvents(
-      Logger eventLogger, ChatCompletionCreateParams request, boolean captureMessageContent) {
+      Context ctx,
```

Comment:
usually we don't use abbreviations so most code uses `context` and `parentContext`

Reply:
Thanks - fixed it